src/webapp/reports/nhwa-attachments/data-attachment-list/FiltersBox.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
This file is exactly the same as `data-comments-list/FiltersBox.ts` and it's unlikely their functionality would diverge as it's a generic component, so it can be moved to another folder and used in both places. Use prop `children`.